src/components/Issues/LatestIssuesCard.tsx

```
@@ -0,0 +1,140 @@
+    case IssueSeverity.MAJOR:
+      return <HighIcon />;
+    case IssueSeverity.MINOR:
+      return <MediumIcon />;
+    case IssueSeverity.INFO:
+      return <LowIcon />;
+    default:
+      return <UnknownIcon />;
+  }
+};
+
+interface LatestIssuesCardProps {}
```

Biome (2.1.2)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)